src/LiveQuery/ParseLiveQueryServer.js

```diff
@@ -93,6 +93,21 @@ class ParseLiveQueryServer {
     }
     this._createSubscribers();
   }
+
+  async handleShutdown() {
```

Comment:
Would it make sense semantically to call this simply `shutdown` instead of `handleShutdown`? For example, `connect` is also not called `handleConnect`.

Reply:
I just saw that all the related handlers of other adapters are called `handleShutdown`. I think that's an argument to use the same name here. If you agree, could you rename this back?

Reply:
Well I think your original argument stands and all the other `handleShutdown` calls should just be transitioned to `shutdown`

Reply:
Alright, let's leave it then.